gravity-forms/gw-disable-submit-button-until-required-fields-are-filled-out.php

```diff
@@ -32,14 +32,15 @@ public function maybe_output_script( $form ) {
 		}
 
 		if ( ! self::$script_output ) {
-			$this->script();
+			add_action( 'wp_footer', array( $this, 'output_script' ) );
+			add_action( 'gform_preview_footer', array( $this, 'output_script' ) );
```

Comment:
Was this to prevent AJAX errors?

Could we use `gform_print_scripts` instead?

Reply:
Yes, and I will check `gform_print_scripts`

Reply:
Doesnt work with `gform_print_scripts`, i think its because it not firing at the correct time.

Reply:
Works for me considering this is a snippet. Thanks for checking!